### PR TITLE
User Data Netplan 

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR updates the cloud-init.md file to add a Netplan network configuration section.

## Review Changes

💻 [Preview URL](https://deploy-preview-1705--docs-spectrocloud.netlify.app/clusters/edge/edge-configuration/cloud-init)

🎫 [PE-2882](https://spectrocloud.atlassian.net/browse/PE-2882)

### CODE REVIEW

File: docs/docs-content/clusters/edge/edge-configuration/cloud-init.md [hunk 1]
```diff
@@ -138,6 +138,48 @@ stages:
                   insecure_skip_verify: true
 ```
 
+
+#### Configure Network With Netplan
+
+You can use the `initramfs` stage and [Netplan](https://netplan.io) to configure network settings before the network initialization. Netplan is a tool that enables you to specify network configurations on Linux systems. Please note that this approach is available for Ubuntu systems only. Refer to the [Netplan Documentation](https://netplan.readthedocs.io/en/stable/) and [Netplan How-to Guides](https://netplan.readthedocs.io/en/stable/examples/) for more information.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/edge/edge-configuration/cloud-init.md:144]:
```suggestion
You can use the `initramfs` stage and [Netplan](https://netplan.io) to configure network settings before the network initialization. Netplan is a tool that enables you to specify network configurations on Linux systems. Note that this approach is available for Linux systems with Netplan installed. Refer to the [Netplan Documentation](https://netplan.readthedocs.io/en/stable/)  for install guidance and the [Netplan How-to Guides](https://netplan.readthedocs.io/en/stable/examples/) for more information.
```

Per our style guide, we don't use please.

**Review by karl-cardenas-coding — APPROVED** [review]
Awesome addition 😄

**Comment by karl-cardenas-coding** [comment]
Backport for 3.4 is handled with #1709

**Comment by caroldelwing** [comment]
Backport for 4.1 handled.

**Pull request merged**